xclim/testing/utils.py

```diff
@@ -666,10 +677,6 @@ def populate_testing_data(
     local_cache : Path
         The path to the local cache. Defaults to the location set by the platformdirs library.
         The testing data will be downloaded to this local cache.
-
-    Returns
-    -------
-    None
```

Comment:
returns a Dataset ? no?

Reply:
Nope, all this function does is fetches data and places it into the cache.